worker/worker-manager/score-worker-manager-impl/src/main/java/io/cloudslang/worker/management/services/InBuffer.java

```diff
@@ -99,8 +106,19 @@
     private void init() {
         capacity = getInteger("worker.inbuffer.capacity", capacity);
         coolDownPollingMillis = getInteger("worker.inbuffer.coolDownPollingMillis", coolDownPollingMillis);
-        logger.info("InBuffer capacity is set to :" + capacity
-                + ", coolDownPollingMillis is set to :" + coolDownPollingMillis);
+
+        startPollingMemoryRatio =
```

Comment:
revert
needToPoll, logMessages, triggerGc, not needed most of it -> use master changes